app/src/main/res/menu/bookmarks_select_multi_not_item.xml

```diff
@@ -7,7 +7,7 @@
     <item
         android:id="@+id/delete_bookmarks_multi_select"
         android:icon="@drawable/mozac_ic_delete"
-        android:iconTint="?primaryText"
+        android:iconTint="@color/white_color"
```

Comment:
I don't see a corresponding change in the code base for this. Why is this one getting changed from primaryText -> white_color?

Reply:
Oops, didn't mean to change this one

Reply:
@NotWoods Did you want to revert this?